engine/script/src/script_table.cpp

```diff
@@ -299,7 +299,7 @@ namespace dmScript
         return total_size;
     }
 
-    uint32_t DoCheckTableSize(lua_State* L, int index)
+    uint32_t DoCheckTableSize(lua_State* L, int index, int total_size)
```

Comment:
Not sure about the name here. accumulated_size? size_of_parent? or buffer_index?

Reply:
Parent is good.

Parent offset?
The parent size isn't done yet 🤔